src/field/phone-number/phone_number_pane.jsx

```diff
@@ -29,13 +28,13 @@ export default class PhoneNumberPane extends React.Component {
         {header}
         <SelectInput
           icon={icon}
-          isValid={!isFieldVisiblyInvalid(lock, "location")}
+          isValid={!c.isFieldVisiblyInvalid(lock, "location")}
           name="location"
           placeholder=""
           label={humanLocation(lock)}
           onClick={() => startOptionSelection(l.id(lock), "location", "", icon)}
         />
-        <PhoneNumberInput ref="phoneNumberInput"
```

Comment:
that part belongs to the passwordless widget. Probable it is not being used as  you said but couldnt tell. I would left it there and add a `//TODO` so when we start working on that widget, we can assess and test what happens with that. makes sense?